sdk/communication/Azure.Communication.Identity/tests/CommunicationIdentityClient/CommunicationIdentityClientLiveTests.cs

```diff
@@ -61,6 +62,32 @@ void ValidateScopesIfNotSanitized()
             }
         }
 
+        [Test]
+        [TestCase(AuthMethod.ConnectionString, TestName = "IssuingTurnCredentialsWithConnectionString")]
+        [TestCase(AuthMethod.KeyCredential, TestName = "IssuingTurnCredentialsWithKeyCredential")]
+        public async Task IssuingTurnCredentialsGeneratesTurnCredentials(AuthMethod authMethod)
+        {
+            CommunicationIdentityClient client = authMethod switch
+            {
+                AuthMethod.ConnectionString => CreateClientWithConnectionString(),
+                AuthMethod.KeyCredential => CreateClientWithAzureKeyCredential(),
```

Comment:
Do we not support managed identity?

Reply:
TURN endpoint only supports HMAC auth, doesn't support BearerToken Authentication. The Managed Identity test in the SDK was sending the token as bearer token and I was getting 'unauthorized'.

Reply:
What is the plan for supporting managed identity, because `CommunicationIdentityClient` can be instantiated using managed identity

Reply:
Looking into this one. Apparently it's supported, but I get 401 unauthorized from the endpoint. Following up if there's something I'm missing with respect to scopes? The TURN endpoint itself uses AuthenticationScheme.Server, which means it should accept both HMAC and AAD auth.

Reply:
Talked to Reza and Andrew offline, will address this in a subsequent PR as this needs to be coordinated with service side updates.